app/component/RouteNumberContainer.js

```diff
@@ -52,6 +56,8 @@ RouteNumberContainer.defaultProps = {
   className: '',
   withBicycle: false,
   occupancyStatus: undefined,
+  externalFeed: false,
+  mode: undefined,
```

Comment:
it does not make sense to pass these two props separately. Mode can already include external postifix.

Reply:
This is now changed